app/models/donation_chart.rb

```diff
@@ -8,12 +8,25 @@ def self.donation_by_date(candidate)
     donation_date = donation_date.to_s.gsub('"', '')
   end
 
+  def self.donation_issues
+    Issue.all.map {|issue| issue.topic }
```

Comment:
Space between { and | missing.<br>Pass `&:topic` as an argument to `map` instead of a block.